app/assets/javascripts/views/users.js

```diff
@@ -0,0 +1,37 @@
+App.Views.Users = Backbone.View.extend({
+  el: 'section',
+  template: JST['templates/users'],
+
+  initialize: function(options) {
+    this.users = options.users;
+
+    this.data = {
```

Comment:
'data' attribute looks redundant here. It might be confusing to use such broad term.